Client/Frontend/Library/HistoryPanel/HistoryPanelViewModel.swift

```diff
@@ -142,12 +142,35 @@ class HistoryPanelViewModel: Loggable, FeatureFlaggable {
     func performSearch(term: String, completion: @escaping (Bool) -> Void) {
         isFetchInProgress = true
 
-        profile.history.getHistory(matching: term,
-                                   limit: searchQueryFetchLimit,
-                                   offset: searchCurrentFetchOffset) { results in
-            self.isFetchInProgress = false
-            self.searchResultSites = results
-            completion(!results.isEmpty)
+        switch self.profile.historyApiConfiguration {
+        case .old:
+            profile.history.getHistory(matching: term,
```

Comment:
(ditto here re indentation, compare with, eg, line 160-164 or so - obviously you should ignore these comments if some formatter decided this is correct)